eyeshade/migrations/0016_geo_referral/up.sql

```diff
@@ -0,0 +1,5 @@
+select execute($$
+
+insert into migrations (id, description) values ('0016', 'geo_referral');
+
```

Comment:
@maikelmclauflin I think we discussed adding a unique constraint on (created_at, country_code) just in case

Reply:
fixed in: dee2446